modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc

```diff
@@ -1,6 +1,8 @@
 [id='rn-new-and-changed-ref']
 = New and changed features
 
+* {PRODUCT} only supports TLS 1.2 in this release.
```

Comment:
like this, I think it might frighten the horses, how about

Support for x, y is removed from {PRODUCT}. Only TLS 1.2 is supported in this release.

Reply:
How about:

> Support for TLS 1.0 and TLS 1.1  is removed from {PRODUCT}. Only TLS 1.2 is supported in this release.